### PR TITLE
Update torbrowser-launcher to 0.3.5

### LINKED ISSUE #21: Tor Browser developers key auto update does not work with Flatpak
Tor Browser Launcher has an auto update mechanism for the Tor Browser developers key, however this mechanism does not seem to work with the Flatpak build. This was already discussed [here](https://github.com/flathub/com.github.micahflee.torbrowser-launcher/issues/14), but I am opening a separate issue for it since it is still a problem.

/cc @micahflee

### PR DESCRIPTION
**Pull request by micahflee**
Fixes #21

### CODE REVIEW

File: com.github.micahflee.torbrowser-launcher.yaml [hunk 1]
```diff
@@ -133,12 +133,11 @@ modules:
     buildsystem: simple
     sources:
       - type: archive
-        url: https://github.com/micahflee/torbrowser-launcher/archive/v0.3.4.tar.gz
-        sha256: 783224924574410755b3fdedb24d3649deeb7bb8207b0acd6e00a66fdb284630
+        url: https://github.com/micahflee/torbrowser-launcher/archive/refs/tags/v0.3.5.tar.gz
+        sha256: b91e632974785d57b567121593aa0238348ca9568025ed71e0f041b960143414
       - type: patch
         paths:
```
Comment (AsciiWolf) [inline com.github.micahflee.torbrowser-launcher.yaml:139]:
```suggestion
        path: use-flatpak-sys-prefix.patch
```

File: com.github.micahflee.torbrowser-launcher.yaml [hunk 2]
```diff
@@ -133,12 +133,11 @@ modules:
...
           - use-flatpak-sys-prefix.patch
```
Comment (AsciiWolf) [inline com.github.micahflee.torbrowser-launcher.yaml:140]:
```suggestion
```

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

**Comment by AsciiWolf** [comment]
Thanks! I see that you forgot to bump the version [in AppStream metadata](https://github.com/micahflee/torbrowser-launcher/blob/develop/share/metainfo/torbrowser.appdata.xml#L32). Any chance you could redo the 0.3.5 tag with this fixed?

**Comment by micahflee** [comment]
Okay, this one should be good I think.

**Comment by AsciiWolf** [comment]
Thanks!

**Pull request merged**